package.json

```diff
@@ -1,6 +1,10 @@
 {
     "name": "dfe-analytics-dataform",
     "dependencies": {
+<<<<<<< HEAD
```

Comment:
@stevenleggdfe you'll need to remove this from the package.json. Looks like a conflict added